api/types/role.go

```diff
@@ -56,6 +56,10 @@ func (f *RoleFilter) Match(role *RoleV6) bool {
 		}
 	}
 
+	if f.SkipSystemRoles {
+		return !IsSystemResource(role)
+	}
+
```

Comment:
This check seems faster then the previous one, should we swap them?

Reply:
Changed the order, added tests.